src/core/server/config/ensure_valid_configuration.ts

```diff
@@ -9,22 +9,27 @@
 import { ConfigService } from '@kbn/config';
 import { CriticalError } from '../errors';
 
+const ignoredPaths = ['dev.', 'elastic.apm.'];
+
+const invalidConfigExitCode = 78;
+const legacyInvalidConfigExitCode = 64;
```

Comment:
Keeping `64` for invalid config keys until `8.0` for BWC. Added a task to change that in the `8.0` meta issue.